web-components/src/components/link/tokens/mdv2-link-tokens.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+
+const colors = require("@momentum-ui/tokens/dist/colors.json");
+
+const link = {
+  prefix: "mdv2",
```

Comment:
This file is an example of an updated component te same will need to be done for other files